impeller/renderer/backend/vulkan/device_buffer_vk.cc

```diff
@@ -3,9 +3,78 @@
 // found in the LICENSE file.
 
 #include "impeller/renderer/backend/vulkan/device_buffer_vk.h"
+#include "fml/logging.h"
 
 namespace impeller {
 
+DeviceBufferAllocationVK::DeviceBufferAllocationVK(
+    const VmaAllocator& allocator,
+    VkBuffer buffer,
+    VmaAllocation allocation,
+    VmaAllocationInfo allocation_info)
+    : allocator_(allocator),
+      buffer_(buffer),
+      allocation_(allocation),
+      allocation_info_(allocation_info) {}
+
+DeviceBufferAllocationVK::~DeviceBufferAllocationVK() {
+  vmaDestroyBuffer(allocator_, buffer_, allocation_);
```

Comment:
Checking here for null arguments or return values (if there is one)?

Reply:
Done! This doesn't have a return value.